doc/reference/indexer.rst

```diff
@@ -149,13 +149,13 @@ Configuration
 List the possible configuration options with ``phpactor config:dump | grep
 indexer``, explanations of some important ones:
 
-- ``indexer.enabled_watchers``: List of watchers to enable (e.g. `inotify`,
+- :ref:`param_indexer.enabled_watchers`: List of watchers to enable (e.g. `inotify`,
   `find`).
-- ``indexer.include_patterns``: List of glob patterns to include
-- ``indexer.exclude_patterns``: List of glob patterns to exclude
-- ``indexer.poll_time``: Poll time used for polling watchers (e.g. ``find``,
-  ``php``
-- ``indexer.buffer_time``: Time to wait to collect batch messages from
+- :ref:`param_indexer.include_patterns`: List of glob patterns to include
+- :ref:`param_indexer.exclude_patterns`: List of glob patterns to exclude
+- :ref:`param_indexer.stub_paths`: List of external paths to index
+- :ref:`param_indexer.poll_time`: Poll time used for polling watchers (e.g. ``find``, ``php``)
+- :ref:`param_indexer.buffer_time`: Time to wait to collect batch messages from
```

Comment:
Are you aware that this documentation is generated?

Reply:
No, it's generated from which other part of the documentation ?

Reply:
https://github.com/phpactor/debug-extension/blob/master/lib/Model/ExtensionDocumentor.php

e.g. https://github.com/phpactor/indexer-extension/blob/master/lib/Extension/IndexerExtension.php#L95-L104

Reply:
The command to generate:

```
phpactor development:configuration-reference
```

Reply:
OK I don't know how I did it but somehow forced push my changed ha closed the PR and marked it as merged ><

I updated the `doc/reference/configuration.rst` file using the command, the PR for the indexer extension: https://github.com/phpactor/indexer-extension/pull/25

Reply:
Cool - in general I would generate this file before release anyway, so no need to update it specifically.